src/main/kotlin/com/jetbrains/micropython/settings/MicroPythonSettingsPanel.kt

```diff
@@ -100,7 +100,7 @@ class MicroPythonSettingsPanel(private val module: Module, disposable: Disposabl
                             )
                             .validationInfo { comboBox ->
                                 val portName = comboBox.selectedItem.asSafely<String>()
-                                if (portName.isNullOrBlank()) ValidationInfo("No port name provided").withOKEnabled()
+                                if (portName.isNullOrBlank()) ValidationInfo("No port name was provided").withOKEnabled()
```

Comment:
_is provided_ sounds better IMO

Reply:
I think that it has to stay "was provided" as the action happened in the past, the user first had to provide a port name and if he did not, then it can be said that "No port name was provided". 

https://textranch.com/c/is-provided-or-was-provided/

Reply:
Looking at your link: 
>"was provided" is used in the past tense to indicate something that was provided at a specific point in the past

We're not talking about any specific point in the past, but about the situation which takes place right now.
So either present simple passive "is provider" or present perfect passive "has been provided". Latter one is too long for the error message.

Reply:
> Looking at your link:
> 
> > "was provided" is used in the past tense to indicate something that was provided at a specific point in the past
> 
> We're not talking about any specific point in the past, but about the situation which takes place right now. So either present simple passive "is provider" or present perfect passive "has been provided". Latter one is too long for a error message.

Grammatically all three options are actually correct "was, is" or leaving it the way it was originally - all are valid grammatically. 

The reason why I suggested using "was" is that this just sounds natural to me when dealing with error messages in general, to list some examples:
	•	“An unexpected error occurred.”
	•	“File not found.”
	•	“Connection was lost.”

Using "is" sounds more like an observation rather than an error message, to me at least.
 
Yes the situation is taking place right now, but looking at the menu, I first had to leave the port name field empty, the IDE had to evaluate it and tell me I have left it empty, meaning "no port name was provided". This is just my intuition, what I expect this kind of error message to look like.

But you have the final say, this is just my perception. Either "was provided" or actually even just "provided" sound fine to me. If you want I can change the commit to "is provided" as well. Thanks for your patience with my PRs!

Reply:
&lt;grammarnazi&gt;
"File not found." is a syntactically incorrect statement, a couple of words is omitted for brevity. It should be "The file is not found", but in it is acceptable in error messages
&lt;/grammarnazi&gt;

So, let it be "provided" or "is provided" - that's up to you.

> Thanks for your patience with my PRs!

On the contrary, I'm glad that I'm not a lone wolf here.

Reply:
> <grammarnazi> "File not found." is a syntactically incorrect statement, a couple of words is omitted for brevity. It should be "The file is not found", but in it is acceptable in error messages </grammarnazi>

I don't think that is correct. The system first searched for the file, it did not find it and then it is letting you know that "The file was not found". if I borrowed a book from the library, didn't return it in time, and then came in to borrow another without returning the first one, the librarian probably wouldn't let me borrow it saying "The first book was not returned", he wouldn't say "The first book is not returned", that would sound odd.

But I have changed the port error message back to "Port name not provided", it stays neutral that way.